### PR TITLE
Regenerate build context using https://github.com/ansible/ansible-builder/pull/157

### PR DESCRIPTION
**Pull request by shanemcd**
Depends-On: https://github.com/ansible/ansible-builder/pull/157

### CODE REVIEW

File: Containerfile [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-RUN ansible-galaxy collection install -r /build/requirements.yml --collections-path /usr/share/ansible/collections
+RUN ansible-galaxy role install -r /build/sources/requirements.yml --roles-path /usr/share/ansible/roles
+RUN ansible-galaxy collection install -r /build/sources/requirements.yml --collections-path /usr/share/ansible/collections
 
 RUN mkdir -p /usr/share/ansible/roles /usr/share/ansible/collections
 
 FROM quay.io/ansible/python-builder:latest as builder
 
-ADD requirements_combined.txt /tmp/src/requirements.txt
-ADD bindep_combined.txt /tmp/src/bindep.txt
```
Comment (pabelanger) [inline Containerfile:13]: This looks to have been deleted

File: Containerfile [hunk 2]
```diff
@@ -19,5 +18,3 @@ FROM quay.io/ansible/ansible-runner:devel
 COPY --from=galaxy /usr/share/ansible/roles /usr/share/ansible/roles
 COPY --from=galaxy /usr/share/ansible/collections /usr/share/ansible/collections
 
-COPY --from=builder /output/ /output/
-RUN /output/install-from-bindep && rm -rf /output/wheels
```
Comment (pabelanger) [inline Containerfile]: same

File: Containerfile (hunk 1 above)
Comment (shanemcd) [reply Containerfile:13]: Oops. This was a git-derp. Fixed.

**Review by AlanCoding — APPROVED** [review]

**Comment by AlanCoding** [comment]
looks like Zuul was using the requirements file

```

2020-12-16 01:51:00.035235 \| ubuntu-bionic \| setting PATH=/home/zuul/src/github.com/ansible/awx-ee/.tox/ansible-builder/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games
--
944 | 2020-12-16 01:51:00.037161 \| ubuntu-bionic \|   /home/zuul/src/github.com/ansible/awx-ee/.tox$ /usr/bin/python3 -m virtualenv --python /usr/bin/python3 ansible-builder >/home/zuul/src/github.com/ansible/awx-ee/.tox/ansible-builder/log/ansible-builder-0.log
945 | 2020-12-16 01:51:05.599657 \| ubuntu-bionic \| ansible-builder installdeps: -r/home/zuul/src/github.com/ansible/awx-ee/requirements.txt
946 | 2020-12-16 01:51:05.599795 \| ubuntu-bionic \| setting PATH=/home/zuul/src/github.com/ansible/awx-ee/.tox/ansible-builder/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games
947 | 2020-12-16 01:51:05.600636 \| ubuntu-bionic \|   /home/zuul/src/github.com/ansible/awx-ee$ /home/zuul/src/github.com/ansible/awx-ee/.tox/ansible-builder/bin/pip install -r/home/zuul/src/github.com/ansible/awx-ee/requirements.txt
948 | 2020-12-16 01:51:06.059520 \| ubuntu-bionic \| ERROR: Could not open requirements file: [Errno 2] No such file or directory: '/home/zuul/src/github.com/ansible/awx-ee/requirements.txt'
```

So if these are requirements to build, is an empty file sufficient?

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
This is failing, because we are using the 'released' version of ansible-builder, but this content was generated using newer version.  I am working on a fix for that now, so we can use depends-on for ansible-builder too.

**Comment by AlanCoding** [comment]
`sources` -> `_build`

**Pull request closed without merging**